sources/osg/FrameBufferObject.js

```diff
@@ -244,6 +255,19 @@ FrameBufferObject.prototype = MACROUTILS.objectInherit( GLObject.prototype, MACR
 
     },
 
+    checkAllowedSize: function ( w, h ) {
+
+        var maxSize = WebglCaps.instance().getWebGLParameter( 'MAX_RENDERBUFFER_SIZE' );
+
+        if ( w === 0 || h === 0 || h > maxSize || w > maxSize ) {
+            Notify.error( 'width or height makes frame buffer not bindable' );
```

Comment:
Maybe we can have more information in the error message:
width (value) or height (value) makes frame buffer not bindable. Max RenderBuffer is 'value'